artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -935,11 +978,14 @@ components:
         application/json:
           schema:
             $ref: "#/components/schemas/ErrorInfo"
-          example:
-            status: 415
-            code: UNSUPPORTED_MEDIA_TYPE
-            message: "The specified media type is not supported"
-    CreateSubscription422:
+          examples:
+            GENERIC_415_UNSUPPORTED_MEDIA_TYPE:
+              description: Payload format of the request is in an unsupported format by the Server. Should not happen
+              value:
+                status: 415
+                code: UNSUPPORTED_MEDIA_TYPE
+                message: The server refuses to accept the request because the payload format is in an unsupported format
+    Generic422:
```

Comment:
I think it should be named specifically, since it contains subscription specific examples

Reply:
hummm so I not get it because when I read in the CAMARA_common.yam that we can have in `Generic422`:
```
            GENERIC_422_{{SPECIFIC_CODE}}:
              description: Any semantic condition associated to business logic, specifically related to a field or data structure
              value:
                status: 422
                code: { { SPECIFIC_CODE } }
                message: { { SPECIFIC_CODE_MESSAGE } }
``
(lines 373) I was thinking this is a placeholder for specific code.  
What is your understanding @akoshunyadi ?

Reply:
I've changed to CreateSubscriptionUnprocessableEntity422 ;)
Hope that fine for @PedroDiez